modules/rosa-deleting-cluster.adoc

```diff
@@ -9,24 +9,109 @@ endif::[]
 
 :_content-type: PROCEDURE
 [id="rosa-deleting-cluster_{context}"]
-= Deleting a cluster
+ifndef::sts[]
+= Deleting a ROSA cluster
+endif::sts[]
+ifdef::sts[]
+= Deleting a ROSA cluster and the cluster-specific IAM resources
+endif::sts[]
 
-You can delete an {product-title} cluster using the `rosa` CLI.
+ifndef::sts[]
+You can delete a {product-title} cluster using the ROSA CLI (`rosa`).
+endif::sts[]
 
 ifdef::sts[]
-You can also use the `rosa` CLI to delete the AWS Identity and Access Management (IAM) account-wide roles, the cluster-specific Operator roles, and the OpenID Connect (OIDC) provider. The cluster deletion must complete before you remove the IAM resources, because the resources are used in the cluster deletion and clean-up processes.
+You can delete a {product-title} (ROSA) with AWS Security Token Service (STS) cluster by using the ROSA CLI (`rosa`) or {cluster-manager-first}.
 
-If add-ons are installed, the deletion takes longer because add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
+After deleting the cluster, you can clean up the cluster-specific Identity and Access Management (IAM) resources in your AWS account by using the ROSA CLI (`rosa`). The cluster-specific resources include the Operator roles and the OpenID Connect (OIDC) provider.
 
 [IMPORTANT]
 ====
-Account-wide IAM roles and policies might be used by other ROSA clusters in the same AWS account. You must only remove the resources if they are not required by other clusters.
+The cluster deletion must complete before you remove the IAM resources, because the resources are used in the cluster deletion and clean-up processes.
 ====
+
+If add-ons are installed, the cluster deletion takes longer because add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
 endif::sts[]
 
+.Prerequisites
+
+* You have installed a ROSA cluster.
+* You have installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your installation host.
```

Comment:
it's not necessary to have the `oc` CLI installed to delete a ROSA cluster.

Reply:
Resolved.